test/integration/targets/iam_user/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: set up aws connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: yes
+
+- name: Test that a user can be created, attached to a group, and then removed
+  block:
+    - name: ensure ansible user exists
+      iam_user:
+        name: AnsibleTestUser
```

Comment:
It would be better here to define the username in a defaults section and use '{{ resource_prefix }}-user' as it reduces the risk of one test run messing with another